Add `3.33.0` changelog (#6885)
terraform-providers/terraform-provider-google: CHANGELOG.md

## 3.33.0 (Unreleased)

DEPRECATIONS:

> * compute: deprecated `enableLogging` on `google_compute_firewall`, define `logConfig.metadata` to enable logging instead. ([#6871](https://github.com/terraform-providers/terraform-provider-google/pull/6871))

Reviewer: ```suggestion
* compute: deprecated `enable_logging` on `google_compute_firewall`, define `log_config.metadata` to enable logging instead. ([#6871](https://github.com/terraform-providers/terraform-provider-google/pull/6871))
```

FEATURES:
* **New Resource:** `google_active_directory_domain` ([#6866](https://github.com/terraform-providers/terraform-provider-google/pull/6866))

> * **New Resource:** `google_dataflow_flex_template_job` ([#6851](https://github.com/terraform-providers/terraform-provider-google/pull/6851))

Reviewer: I think this is beta-only

IMPROVEMENTS:
* cloudrun: added `ports` field to `google_cloud_run_service` `templates.spec.containers` ([#6873](https://github.com/terraform-providers/terraform-provider-google/pull/6873))

> * compute: added support to `google_compute_backend_service` for setting a network endpoint group as `backend.group` ([#6853](https://github.com/terraform-providers/terraform-provider-google/pull/6853))

Reviewer: @slevenick is this part of this release or the previous one (I thought I remembered you saying it was in 3.32.0)

Reviewer 2: Yup, I patched this into 3.32.0 so it can be removed from this changelog. https://github.com/terraform-providers/terraform-provider-google/blob/master/CHANGELOG.md

> * compute: added support to `google_compute_backend_service` for setting a network endpoint group as `backend.group` ([#6853](https://github.com/terraform-providers/terraform-provider-google/pull/6853))

Reviewer 2: ```suggestion

```

> * compute: added `logConfig.metadata` to `google_compute_firewall`, defining this will enable logging. ([#6871](https://github.com/terraform-providers/terraform-provider-google/pull/6871))

Reviewer: ```suggestion
* compute: added `log_config.metadata` to `google_compute_firewall`, defining this will enable logging. ([#6871](https://github.com/terraform-providers/terraform-provider-google/pull/6871))
```
